src/person/models.py

```diff
@@ -22,4 +24,5 @@ class Person(models.Model):
     locations = models.ManyToManyField(Location, default=None, blank=True)
     interests = models.ManyToManyField(Interest, default=None, blank=True)
     groups = models.ManyToManyField(Group, default=None, blank=True)
-    has_onboarded = models.BooleanField(default=None, null=True)
+    has_onboarded = models.BooleanField(default=False)
```

Comment:
for future reference, technically these should be `null=True` since if we have people already in the system then we'll get an error because we have Person instances without this field, so it must be null, but `null=False` by default so we get a non-nullable error. however, for this case, it's fine because i can just delete all the users since we're testing anyway. just want you to keep this in mind for the future for when we add new properties but can't just willy nilly delete all users lol